### DISCUSSION TITLE
Blur makes the table very dark

### DISCUSSION THREAD

**Discussion by mrclrchtr** [icflorescu/mantine-datatable#541] [Q&A]
Hi,

Blur makes the table very dark. This is a clear difference to the light mode. Is that intentional?  Can I adjust it somehow?

![image](https://github.com/icflorescu/mantine-datatable/assets/16151098/789b9448-fc3a-422f-8d08-16d22b34ae41)

https://icflorescu.github.io/mantine-datatable/examples/asynchronous-data-loading/

**Comment by icflorescu** [comment]
Thanks for bringing this to my attention, @mrclrchtr!
Too dark indeed, now that you mentioned it. Will fix it in the next release today.

**Reply by mrclrchtr** [reply]
Thank you very much! 😊
